homeassistant/components/plex/media_player.py

```diff
@@ -125,52 +159,45 @@ def _clear_media_details(self):
         # Clear library Name
         self._app_name = ""
 
-    def refresh(self, device, session):
+    def update(self):
         """Refresh key device data."""
         self._clear_media_details()
 
-        if session:  # Not being triggered by Chrome or FireTablet Plex App
-            self._session = session
-        if device:
-            self._device = device
+        self._available = self.device or self.session
```

Comment:
Can `self.device` be `None`?

Reply:
As above, no. I guess this can be simplified.

Reply:
To clarify, `device` should never `None` unless `session` is also `None`.

Reply:
Actually, I'm not 100% this is going to be true in all cases. I supposed there's a possibility where a `session` object doesn't actually contain anything in its `players` list attribute. I think this should remain as-is for now.